### ISSUE TITLE
emacs integration

### ISSUE THREAD

**Issue by bijang** [stumpwm/mahogany#13]
this is an echo from the stumpwm wishlist, I personally dont have enough experience with exwm to know if there is more that can be achieved, I believe things like org agenda integration with calendar perhaps? 
Perhaps other people could elaborate, I'm sure there is no end to the possible integration between the two program

one idea I have was, is it possible to org campture the destop setup (ie what groups and frames were open a particular moment, so that when you restart, you can immediately open all programs very quickly?

[please feel free to close this issue if it inappropriate]

**Comment by minikN** [comment]
I guess the frontend part (which is yet to be written) could be written in elisp instead of cl. That way it could replace exwm entirely. Easier said than done I guess.

**Comment by sdilts** [comment]
I don't think we should explicitly support this; if you want to manipulate mahogany through emacs, use the standardized protocols for requesting changes to a wayland compositor.
